.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
```

Comment:
What is directory here? `/` doesn't seem like a sensible value, surely it should be `.github` or something?

Reply:
😕 This confuses me too but it seems to be the supported path for actions:

> The `directory` must be set to `"/"` to check for workflow files in `.github/workflows`.